site/layouts/shortcodes/card.html

```diff
@@ -0,0 +1,60 @@
+{{- /*
+  Usage: `card args`
+
+  `args` are all optional and can be one of the following:
+    * border: Add some borders to the card - default = false
```

Comment:
For readability, please reorder them alphabetically.
Maybe rename to "borders" since it add borders to the card.